### PR TITLE
refactor: add raw call to GetMarketDataHistoryByIDRequest

### PR DESCRIPTION
**Pull request by witgaw**

### CODE REVIEW

File: tests/vega_sim/api/test_data_raw.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import grpc
 import pytest
 from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime
```
Comment (TomMcL) [inline tests/vega_sim/api/test_data_raw.py]: Could you change these to just `import datetime`? It's the pattern we've used elsewhere so good to keep consistent :)

**Review by TomMcL — COMMENTED** [review]
Other than the import change, looks good. Would you mind creating an issue for taking this through `data.py` and adding an interface within `service.py`?

**Comment by witgaw** [comment]
> Other than the import change, looks good. Would you mind creating an issue for taking this through `data.py` and adding an interface within `service.py`?

Sure https://github.com/vegaprotocol/vega-market-sim/issues/325

File: tests/vega_sim/api/test_data_raw.py (hunk 1 above)
Comment (witgaw) [reply tests/vega_sim/api/test_data_raw.py]: done now

**Review by TomMcL — APPROVED** [review]
LGTM

**Pull request merged**